<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="paintArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupMove">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>190</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Move</string>
    </property>
    <widget class="QLabel" name="labelMoveX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dX</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMoveY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dY</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMoveZ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dZ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMoveX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMoveY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMoveZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonMove">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupRotate">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>190</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotate</string>
    </property>
    <widget class="QLabel" name="labelRotateX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>oX</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRotateY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>oY</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRotateZ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>oZ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRotateX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>angle°</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRotateY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>angle°</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRotateZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>angle°</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRotate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupScale">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>190</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Scale</string>
    </property>
    <widget class="QLabel" name="labelScale">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kX</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScaleY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kY</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScaleZ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kZ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditScaleX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditScaleY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditScaleZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonScale">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Open file</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionOpenFile_2">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
